app-load-params-db.json

```diff
@@ -1733,7 +1733,7 @@
     "path": ["44'/330'"]
   },
   "tezos_baking": {
-    "appFlags": {"nanos": "0x800", "nanos2": "0xa40", "nanox": "0xa40", "stax": "0xa40"},
+    "appFlags": {"nanos": "0x040", "nanos2": "0x040", "nanox": "0x040", "stax": "0x040"},
```

Comment:
Their won't be BLE support on NanoX and Stax? If you want it, you need 0200 flag on them.

Reply:
Oh yes, of course!
Thank you so much!

Reply:
We do not wish to support `BLE` for _NanoX_ and _Stax_, especially for the Baking app, as we really want the devices to remain plugged in.